packages/cli/src/commands/build.ts

```diff
@@ -66,25 +64,27 @@ export const build: Command = {
         `-c, --client-config <${intlMsg.commands_common_options_configPath()}>`,
         `${intlMsg.commands_common_options_config()}`
       )
-      .option(`-n, --no-codegen`, `${intlMsg.commands_build_options_n()}`)
+      .option(`-n, --skip-codegen`, `${intlMsg.commands_build_options_n()}`)
```

Comment:
`-n` seems a bit unnatural here now…

Usually, when the 1st character is already taken, the next available character is taken.

For options that are not commonly used, I have often seen the practice of even completely omitting the one-character option, and leaving only the long version.

Maybe you might want to consider one of these?

So either `-k` or no single-char option at all here?

This is just my opinion though, feel free to ignore.

Reply:
Resolved, reverted back to `--no-codegen`